torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -158,28 +160,31 @@ c10::IValue getFunctionTuple(
   // register size
   auto register_size = static_cast<int>(code.register_size());
 
-  // module debug info
-  std::vector<IValue> module_paths;
+  auto table = Table({{"instructions", Tup(instructions)},
+                      {"operators", Tup(operators)},
+                      {"constants", Tup(constants)},
+                      {"types", Tup(types)},
+                      {"register_size", register_size}});
+  elements.push_back(Tup({func.qualname().qualifiedName(), table}));
+
   if (save_debug_info_in_bytecode) {
+    // module debug info
+    std::vector<IValue> module_paths;
     module_paths.reserve(op_module_paths.size());
     for (auto& path : op_module_paths) {
       module_paths.emplace_back(std::move(path));
     }
+    auto module_debug_info = Table({{"module_debug_info", Tup(module_paths)}});
+    debug_info_elements.push_back(
+        Tup({func.qualname().qualifiedName(), module_debug_info}));
```

Comment:
This is good that you are adding association information between bytecode.pkl and debug_info.pkl